src/rgw/rgw_swift_auth.h

```diff
@@ -152,6 +164,7 @@ class SwiftAnonymousEngine : public rgw::auth::AnonymousEngine {
 
 public:
   SwiftAnonymousEngine(CephContext* const cct,
+                       const SwiftAnonymousApplier::Factory* const apl_factory,
                        const rgw::auth::LocalApplier::Factory* const apl_factory,
```

Comment:
hrm. not sure why the apl_factory is now in twice, I swear it wasn't when build it. Anyway easy fix.